client/src/components/Login/Login.tsx

```diff
@@ -1,6 +1,7 @@
-import React, {useState} from 'react'
+import React from 'react'
 import './Login.css'
 
+const SPOTIFY_URL = "https://accounts.spotify.com/authorize?client_id=545bc37c391746368c3e07cef94bfd51&response_type=code&redirect_uri=http://localhost:3000&scope=streaming%20streaming%20user-read-email%20user-read-private%20user-library-read%20user-library-modify%20user-read-playback-state%20user-modify-playback-state"
```

Comment:
Consider storing the client_id to a env variable.  [dotenv](https://www.npmjs.com/package/dotenv)

Reply:
client_id should be fine, its the id for our app which will be public. There's a secret key as well that I'll be storing with dotenv